### ISSUE TITLE
What is current release? 4.2 tagged and updated in wp-cli but wp repo is at 4.1.1

### ISSUE THREAD

**Issue by chuckreynolds** [Automattic/jetpack#4751]
WP-cli said there was an update the other day so I `wp plugin update jetpack` which pulled and installed 4.2.

Some login redirect issues happening and looking at things that changed recently... 4.2 was released on the [releases page](https://github.com/Automattic/jetpack/releases) but after a RC4 was tagged as pre-release. 
SOOO... ??

what's up?

**Comment by ebinnion** [comment]
After releasing 4.2 yesterday and getting reports of errors, we have rolled back stable to 4.1.1. We are currently working out the issues and will be releasing 4.2.1.

**Comment by chuckreynolds** [comment]
WTF!?  guys... come on.

**Comment by kraftbj** [comment]
The particular issue that led to the rollback was specifically impacting multisites with multiple millions of users. While we tested the release on a single site with 1.5M users and multisites of more typical sizes, we didn't on a multisite with millions of users.

We wanted to reduce the risk of another site of that size being impacted, so we opted to be conservative and rollback the stable tag.

After the rollback, we did see some reports of login issues that appear in specific circumstances with a specific caching setup/plugin. With those reports in hand, we opted to further delay 4.2.1 until we could determine that issue and resolve it. We've duplicated the login issue, have a patch, and completing multiple rounds testing of it now.

With each release, we look at issues that arose and incorporate changes into our quality process to prevent similar issues. In this case, we identified two weaknesses—not testing on a multisite with an user base in the millions and the particular setup that led to the login issues—and will be more mindful of those situations in the future.
